index.js

```diff
@@ -0,0 +1,108 @@
+window.addEventListener("load", () => {
+  const numButtons = document.querySelectorAll(".wButton");
+  const opButtons = document.querySelectorAll(".rButton");
+  const eqButton = document.querySelector(".eButton");
+  const numDisplay = document.querySelector(".numDisplay");
+  const opArray = ["add", "subtract", "multiply", "divide"];
+  let currentString = numDisplay.textContent;
+
+  let currentNum = Number(currentString);
+  let firstNum = "";
+  let currentOp = "";
+  let memNum = "";
+
+  for (let n = 0; n <= 9; n++) {
+    numButtons[n].addEventListener("click", () => {
+      if (currentString.length <= 10) {
+        numDisplay.textContent = currentString + numButtons[n].textContent;
+        currentString = numDisplay.textContent;
+        currentNum = Number(currentString);
+      } else {
+        return;
+      }
+    });
+  }
+
+  numButtons[10].addEventListener("click", () => {
+    if (currentString.includes(".")) {
+      return;
+    } else {
+      numDisplay.textContent = currentString + numButtons[10].textContent;
+      currentString = numDisplay.textContent;
+      currentNum = Number(currentString);
+    }
+  });
+
+  numButtons[11].addEventListener("click", () => {
+    if (currentString.includes("-")) {
+      currentNum = Number(currentString) * -1;
+      numDisplay.textContent = currentNum.toString();
+      currentString = numDisplay.textContent;
+    } else {
+      numDisplay.textContent = "-" + currentString;
+      currentString = numDisplay.textContent;
+      currentNum = Number(currentString);
```

Comment:
Multiplying by -1 would work whether it's positive or negative, so I don't think you need this if statement. You could just do the things inside the initial if statement.

However, you may also need to consider what would happen if you press this minus button immediately after pressing the decimal point, in which case currentString would end in "."